docs/model_zoo.md

```diff
@@ -13,6 +13,26 @@ You can replace `https://s3.ap-northeast-2.amazonaws.com/open-mmlab` with `https
 - For fair comparison with other codebases, we report the GPU memory as the maximum value of `torch.cuda.max_memory_allocated()` for all 8 GPUs. Note that this value is usually less than what `nvidia-smi` shows.
 - We report the inference time as the total time of network forwarding and post-processing, excluding the data loading time. Results are obtained with the script [benchmark.py](https://github.com/open-mmlab/mmdetection/blob/master/tools/analysis_tools/benchmark.py) which computes the average time on 2000 images.
 
+## ImageNet Pretrained Models
+
+It's common to initialize from backbone models pre-trained on ImageNet classification tasks. All pre-trained  model weight links can be found at [open_mmlab](https://github.com/open-mmlab/mmcv/blob/master/mmcv/model_zoo/open_mmlab.json). The following commonly used backbone models are available:
+
+- [ResNet50](https://download.pytorch.org/models/resnet50-19c8e357.pth):  from torchvision's ResNet-50.
+- [ResNet50 Caffe](https://download.openmmlab.com/pretrain/third_party/resnet50_msra-5891d200.pth):  converted copy of [MSRA's original ResNet-50](https://github.com/KaimingHe/deep-residual-networks) model.
```

Comment:
May add the link of its source pkl file [here](https://github.com/facebookresearch/detectron2/blob/master/MODEL_ZOO.md#imagenet-pretrained-models).